.changeset/config.json

```diff
@@ -12,8 +12,7 @@
       "@module-federation/runtime-tools",
       "@module-federation/managers",
       "@module-federation/manifest",
-      "@module-federation/dts-plugin",
-      "@module-federation/native-federation-typescript"
```

Comment:
This change is just temporary to allow the version rollback, right?

Reply:
got it, tysm

Reply:
Ok, but as `native-federation-typescript` has been published with a rollbacked version (0.1.2 if I remember correctly), are you sure that you need to remove it?
I think that you still need it here, because a version 0.5.0 needs to be published now to keep up with the latest changes. Am I missing something?

Reply:
This version of the package will be rereleased to 0.5.0 after this merge so there should be no problems